examples/helm-template/README.md

```diff
@@ -15,8 +15,131 @@ kpt fn run helm-template/local-configs --mount type=bind,src=$(pwd)/helm-templat
 
 ## Expected result
 
+Checking the contents of the `local-configs` directory should reveal the following new yaml files:
+
+```sh
+./local-configs:
+deployment_chart-helloworld-chart.yaml  pod_chart-helloworld-chart-test-connection.yaml  service_chart-helloworld-chart.yaml
+fn-config.yaml                          serviceaccount_chart-helloworld-chart.yaml
+```
+
 Verify the expanded configuration:
 
 ```sh
 kpt cfg cat local-configs
 ```
+
+The expected output should match the following:
```

Comment:
Is this necessary given output of `kpt cfg tree`?

Reply:
Changed to instead show dry run output instead of written output.